src/chaotic.sh

```diff
@@ -39,7 +39,7 @@ CAUR_GPG_PATH="/usr/bin/gpg"
 CAUR_LIB="${CAUR_PREFIX}/lib/chaotic"
 CAUR_GUEST="${CAUR_LIB}/guest"
 CAUR_LOWER_DIR="${CAUR_CACHE}/lower"
-CAUR_LOWER_PKGS=(base base-devel)
+CAUR_LOWER_PKGS=(base base-devel git)
```

Comment:
I disagree with dirtying all the builder's lowerstrap. Which will dirty all containers.
We used to do exactly this in the past and after much trouble successfully removed it.

Reply:
Resolved